csm/core/controllers/information.py

```diff
@@ -77,3 +77,133 @@ async def post(self):
             Log.error(f"Error in checking compatability: {e}")
             raise CsmInternalError(f"{e}")
         return CsmResponse(response)
+
+@CsmAuth.public
+@CsmView._app_routes.view("/api/v2/system/topology")
+class TopologyView(CsmView):
+    """
+    Complete deployment topology will be provided
+    GET: Complete deployment topology
+    """
+    def __init__(self, request):
+        super().__init__(request)
+        self._service = self.request.app[const.INFORMATION_SERVICE]
+
+    async def get(self):
+        """GET REST implementation for complete cortx topology."""
+        Log.info("Fetching cortx topology.")
+        # Call Cortx Information Service
+        response = await self._service.get_topology()
+        return CsmResponse(response)
+
+@CsmAuth.public
+@CsmView._app_routes.view("/api/v2/system/topology/{resource}")
+class ResourcesTopologyView(CsmView):
+    """
+    Get information about all resources from deployment topology.
+    GET: Get information about all resources from deployment topology.
+    """
```

Comment:
Doc is string is duplicated. Please fix it.

Reply:
removed